docs/migrating/incompatibilities.rst

```diff
@@ -14,24 +14,33 @@ The `Cmd.emptyline()
 <https://docs.python.org/3/library/cmd.html#cmd.Cmd.emptyline>`_ function is
 called when an empty line is entered in response to the prompt. By default, in
 cmd_ if this method is not overridden, it repeats and executes the last
-nonempty command entered.  However, no end user we have encountered views this
-as expected or desirable default behavior.  Thus, the default behavior in
-``cmd2`` is to simply go to the next line and issue the prompt again.  At this
-time, ``cmd2`` completely ignores empty lines and the base class
-cmd.emptyline() method never gets called and thus the emptyline() behavior
-cannot be overridden.
+nonempty command entered. However, no end user we have encountered views this
+as expected or desirable default behavior. ``cmd2`` completely ignores empty
+lines and the base class ``cmd.emptyline()`` method never gets called and thus
+the empty line behavior cannot be overridden.
 
 
 Cmd.identchars
 --------------
 
 In cmd_, the `Cmd.identchars
 <https://docs.python.org/3/library/cmd.html#cmd.Cmd.identchars>`_ attribute
-contains the string of characters accepted for command names.  Since version
-0.9.0, ``cmd2`` has ignored ``identchars``. cmd_ uses those characters to split
-the first "word" of the input, but the parsing logic in ``cmd2`` parses on
-whitespace.  This has the added benefit of full unicode support, unlike cmd_ or
-versions of ``cmd2`` prior to 0.9.0.
+contains the string of characters accepted for command names. cmd_ uses those
+characters to split the first "word" of the input, without requiring the user
+to type a space. For example, if ``identchars`` contained a string of all
+alphabetic characters, the user could enter a command like ``L20`` and it would
+be interpreted as the command ``L`` with the first argument of ``20``.
+
+Since version 0.9.0, ``cmd2`` has ignored ``identchars``; the parsing logic in
+``cmd2`` splits the command and arguments on whitespace. While cmd_ technically
```

Comment:
> While cmd_ technically supports unicode, but using non-ascii unicode characters in command names while
simultaneously using ``identchars`` functionality is non-trivial.

This sentence is a bit awkward.